docs/src/reference/scripting.md

```diff
@@ -81,6 +81,19 @@ It explains #name.
 Sum is #add(2, 3).
 ```
 
+Let bindings can also be used to unpack arrays. Note that the underscore `_`
+is special in these assignments, as it can be used multiple times in the
+same assignment.
+
+```example
+#let (x, y) = (1, 2)
+The coordinates are #x, #y.
+
+#let (_, c1, .._, c2) = "Hello World!".clusters()
```

Comment:
would be nicer if `.._` could be just `..` I think. the same also applies to argument lists, where it currently doesn't work to but I guess that's my fault.

Reply:
by the way, this is really cool. I think argument sinks in the middle like this destructuring sink are actually broken because I couldn't be bothered to implement them properly...